src/main/java/techcourse/myblog/persistence/LoginRepository.java

```diff
@@ -0,0 +1,9 @@
+package techcourse.myblog.persistence;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import techcourse.myblog.domain.User;
+
+public interface LoginRepository extends JpaRepository<User, Long> {
```

Comment:
REPOSITORY는 ENTITY를 기준으로 만드는 것이 좋아요.

```suggestion
public interface UserRepository extends JpaRepository<User, Long> {
```